gdk-pixbuf/src/subclass/pixbuf_animation.rs

```diff
@@ -141,8 +141,12 @@ unsafe extern "C" fn animation_get_size<T: PixbufAnimationImpl>(
     let imp = instance.imp();
 
     let (width, height) = imp.size();
```

Comment:
Maybe return directly without calling `size()` if both are `NULL`?

Looks good to me otherwise